src/domain/users-ips/index.ts

```diff
@@ -1,11 +1,14 @@
-export const parseIps = (
-  headers: null | NodeJS.Dict<string | string[]>,
-): IpAddress | null => {
-  if (!headers) return null
+import { ExpressContext } from "apollo-server-express"
```

Comment:
the domain should not depend on apollo-server-express. the logic around `    process.env.NODE_ENV === "development"
      ? context.req?.ip
      : context.req?.headers["x-real-ip"]` should be in the server I think.
      
      @bodymindarts ?

Reply:
@bodymindarts

Reply:
Thats right - domain is purely functional without external dependencies

Reply:
@nicolasburtey I've moved the dependency out of the domain and back to the server.